src/components/__tests__/EditVocabulary.test.tsx

```
@@ -0,0 +1,695 @@
+
+      await user.click(screen.getByRole('button', { name: /Update Vocabulary/i }));
+    };
+
+    test('successfully updates a vocabulary', async () => {
+      const user = userEvent.setup();
+
+      const mockVocabUpdateFn = jest.fn().mockResolvedValue({ error: null });
+      const mockWordDeleteFn = jest.fn().mockResolvedValue({ error: null });
+      const mockWordInsertFn = jest.fn().mockResolvedValue({ error: null });
+
+      // @ts-ignore
```

ESLint

[error] Use "@ts-expect-error" instead of "@ts-ignore", as "@ts-ignore" will do nothing if the following line is error-free.

(@typescript-eslint/ban-ts-comment)